core/controllers/payload_validator.py

```diff
@@ -59,7 +62,7 @@ def validate(handler_args, handler_args_schemas, allowed_extra_args):
                 errors.append('Missing key in handler args: %s.' % arg_key)
                 continue
         try:
-            normalized_value[arg_key] = schema_utils.normalize_against_schema(
+            normalized_value[arg_key] = schema_utils.normalize_against_schema(# type: ignore[no-untyped-call]
```

Comment:
Minor: Here and elsewhere, should there be a comma before each '#' (to separate it from the main code)?

Reply:
Fixed this and other places too.